templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -32,13 +32,6 @@ if [[ "$TEST" = "docs" ]]; then
   make PULP_URL="$PULP_URL" diagrams html
   tar -cvf docs.tar ./_build
   cd ..
-{%- if check_openapi_schema %}
```

Comment:
you can drop it from here: https://github.com/pulp/plugin_template/blob/main/plugin-template#L28

Reply:
good idea!